public_dropin_notebook_environments/python38_snowflake_notebook/agent/cgroup_watchers.py

```diff
@@ -51,7 +51,13 @@ def __init__(self) -> None:
         self._cpuacct_usage_file = cgroup_cpuacct_dir / "cpuacct.usage"
 
     def memory_usage_in_bytes(self) -> int:
-        return self._read_metric(self._memory_usage_file)
+        memory_stat_str = self._memory_usage_file.read_text()
+        total_rss_str = next(
+            iter([stat for stat in memory_stat_str.split("\n") if stat.startswith("total_rss")]),
+            "0",
+        )
+        total_rss = int(total_rss_str.split(" ")[-1])
```

Comment:
nit: I'd recommend using `split` without arguments, thus it splits the string at whitespace characters (spaces, tabs, etc).
Same for all there corresponding places in this PR.

Reply:
Thanks for taking a look - I'll defer to @tpadr in order for us to keep it as close as possible to the existing code in our notebooks repo - code seen here:

https://github.com/datarobot/notebooks/pull/2854